app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copy, file_name, new_file_name = command.split()
```

Comment:
bad name for command variable, use more descriptive name, f.e. command

Reply:
Can't get the idea. I mean we get some command, which we split on command + name + name,which name should be good?